### DISCUSSION TITLE
rb_to_p_superiority()

### DISCUSSION THREAD

**Discussion by SandraJGeiger** [easystats/effectsize#553] [Q&A]
Dear all, 

I calculated one-sample Wilcoxon signed-rank tests and obtained the rank biserial correlation using the rcompanion::wilcoxon.test() function. This function takes ranks and signs into account when calculating the rb correlation. 
Now, for an easier interpretation, I would like to transform the rb correlation with the "rb_to_p_superiority()" function. I was wondering whether this function is only valid for rb correlations that only take the sign into account (x participants below 0 and y participants above 0) or also for rb correlations that weigh the sign and ranks as in the rcompanion package? 

I hope I explained my question sufficiently. 
Many thanks in advance. 
Sandra

**Comment by mattansb** [comment]
Hi Sandara,

I'm not sure which effect size you're using from `{rcompanion}` (there seem to be several), but you can use `{effectsize}` for all of these steps (with CIs!):

``` r
X <- c(1,2,3,3,3,3,4,4,4,4,4,5,5,5,5)

effectsize::rank_biserial(X, mu = 3)
#> r (rank biserial) |       95% CI
#> --------------------------------
#> 0.62              | [0.06, 0.88]
#> 
#> - Deviation from a difference of 3.

effectsize::p_superiority(X, mu = 3, parametric = FALSE)
#> Pr(superiority) |       95% CI
#> ------------------------------
#> 0.81            | [0.53, 0.94]
#> 
#> - Non-parametric CLES

effectsize::rb_to_p_superiority(0.62)
#> [1] 0.81
```

<sup>Created on 2023-01-18 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>
